server/client.go

```diff
@@ -1447,7 +1442,8 @@ func (c *client) flushOutbound() bool {
 	// "nb" will be reset back to its starting position so it can be modified
 	// safely by queueOutbound calls.
 	c.out.wnb = append(c.out.wnb, collapsed...)
-	orig := append(net.Buffers{}, c.out.wnb...)
+	var _orig [1024][]byte
+	orig := append(_orig[:0], c.out.wnb...)
```

Comment:
Where is this used?

Reply:
L1479-1485 to return consumed buffers to the pool. The `net.Buffers.WriteTo()` function modifies the receiver slice by trimming sent buffers off the beginning, so we need to keep a copy of those references to recycle them.